src/drivers/mct/cime_config/testdefs/testlist_drv.xml

```diff
@@ -23,6 +23,15 @@
       </machine>
     </machines>
   </test>
+  <test name="ERS_C3" grid="f19_g17_rx1" compset="A">
```

Comment:
I think that you might be confusing this with anther test, ERS tests restart capability and works in an A case for either N3 or C3.